app/uploaders/image_uploader.rb

```diff
@@ -31,6 +31,16 @@ def store_dir
   #   # do something
   # end
 
+  after :store, :store_binary
+  process :resize_to_fit => [400, 400]
+
+
+  def store_binary file
+     @model.image_binary = Base64.encode64(File.open(@model.image.file.file).read)
+     @model.image_content_type = self.content_type
+     @model.save
```

Comment:
`save!` will throw if anything goes wrong - you might want that. Better an error you can debug than a mysterious one you can't find.